scripts/release_binary.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+VERSION="$1"
+USER="$2"
+TOKEN="$3"
+
+cd /tmp
+mkdir php-ovh-bin
+cd php-ovh-bin
+
+php -r "readfile('https://getcomposer.org/installer');" > composer-setup.php
+php -r "if (hash('SHA384', file_get_contents('composer-setup.php')) === '7228c001f88bee97506740ef0888240bd8a760b046ee16db8f4095c0d8d525f2367663f22a46b48d072c816e7fe19959') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;"
```

Comment:
Seems not very future proof. Can you fix the version number above or aquire the cksum dynamically?